torch/csrc/profiler/kineto_client_interface.cpp

```diff
@@ -0,0 +1,57 @@
+#ifdef USE_KINETO
+#include <libkineto.h>
+#include <caffe2/torch/csrc/api/include/torch/torch.h>
```

Comment:
Not sure how this change even got thru CI, as this header path clearly does not exists

Reply:
Actually, CI was red across the board for this change(see https://github.com/pytorch/pytorch/actions/runs/2192736184), why the heck was it landed then?